common/src/auth/models/response.ts

```diff
@@ -0,0 +1,12 @@
+import { AbstractUser } from '../../user';
+
+export class AuthResponse<User extends AbstractUser> {
+  public token: string;
+  public user: User;
+  public ttl?: number;
+  public refresh_ttl?: number;

```

Comment:
would be better to use class-transformer here

Reply:
@astorozhevsky I have removed `ttl` and `refresh_ttl` fields from this model.